Document that converting from darcs works only for the old format.

Index: mercurial-1.1.2/hgext/convert/__init__.py
===================================================================
--- mercurial-1.1.2.orig/hgext/convert/__init__.py
+++ mercurial-1.1.2/hgext/convert/__init__.py
@@ -18,7 +18,7 @@
     Accepted source formats [identifiers]:
     - Mercurial [hg]
     - CVS [cvs]
-    - Darcs [darcs]
+    - Darcs [darcs] (legacy Darcs 1 format only)
     - git [git]
     - Subversion [svn]
     - Monotone [mtn]
